blog/2023-09-22-documenting-things-posit-conf/index.qmd

```diff
@@ -0,0 +1,128 @@
+---
+title: "Documenting things & Posit Conference Debrief"
+author:
+  - name: Julie Lowndes
+    orcid: 0000-0003-1682-3872
+date: 2023-08-17
+citation: 
+  url: https://openscapes.org/blog/2023-09-22
+categories: [how-we-work, blog]
+#image: 
+---
+
+*Last week I've been at #PositConf2023 in Chicago, reconnecting with the open data science community through folks from RLadies, Posit, and the greater #rstats and python community. I gave a talk, sharing about documenting things and how intentional approaches is helping teams tackle hard challenges and change organizational culture. This post is a light debrief of highlights from the Conference, as well as a short overview of my talk.*
+
+*Quick links:*
+
+-   [*Posit Conf YouTube*]() - coming soon with all talks from the Conference
+- [Documenting things: openly for future us](https://openscapes.github.io/documenting-things) - my slides
+
```

Comment:
- [Blog summary of the conference by Millie Symns!](https://milliesymns.com/blog/2023-09-25-spoke-at-posit-conf-2023/)

Reply:
Just read this and - Ah I had meant to make that a commit not a comment, but if you could please add @stefaniebutland, thank you!